content/hi/docs/reference/glossary/workload.md

```diff
@@ -0,0 +1,21 @@
+---
+title: कार्यभार (Workload)
+id: workloads
+date: 2019-02-13
+full_link: /docs/concepts/workloads/
+short_description: >
+   कार्यभार एक ऐप्लिकेशन है जो कुबेरनेट्स पर चल रहा है।
+
+aka: 
+tags:
+- fundamental
+---
+   कार्यभार एक ऐप्लिकेशन है जो कुबेरनेट्स पर चल रहा है।
+
+<!--more--> 
+
+विभिन्न कोर ऑब्जेक्ट्स जैसे डेमनसेट, डिप्लॉयमेंट, जॉब, रेप्लिकासेट और स्टेटफुलसेट एक कार्यभार के विभिन्न प्रकारों या भागों को प्रतिनिधित्व करते हैं।
```

Comment:
विविध मुख्य वस्तुएं एक कार्यभार के विविध प्रकारों अथवा भागों का प्रतिनिधित्व करती हैं। जैसे डेमनसेट, डिप्लॉयमेंट, जॉब, रेप्लिकासेट और स्टेटफुलसेट।

Reply:
@aj11anuj wouldn't it be more specific if we use `कोर ऑब्जेक्ट्स` rather than `मुख्य वस्तुएं`
it could change the actual meaning of k8s objects

Reply:
Hmm.. actually at that time i was finding better replacement for "Core Objects" in hindi, but didn't found much suitable replacement. So, i tried suggesting "Mukhy Vastue" to see, if any other reviewer also finds it suitable or not. "Core Objects" written in hindi is looking odd to me, Can we try writing "Core Objects" in english as it is or "Core Objects" written in hindi looks odd to me only?